perun-cli/Perun/beans/HashedGenData.pm

```diff
@@ -124,11 +124,32 @@ sub getResourceIds ()
 	return @resourceIds;
 }
 
+sub getVosIds ()
```

Comment:
Even if this is probably the better way, I would change that to 'getVoIds' because other methods are using a singular object for the name instead of plural.

Reply:
Fixed.